TestModels/SimpleTypes/SimpleBoolean/runtimes/rust/tests/simple_boolean_test.rs

```diff
@@ -0,0 +1,49 @@
+use simple_boolean::*;
+/*
+method{:test} GetBoolean(){
+      var client :- expect SimpleBoolean.SimpleBoolean();
+      TestGetBoolean(client);
+      TestGetBooleanSingleValue(client);
+  }
+*/
+
+/*method TestGetBoolean(client: ISimpleTypesBooleanClient)
+{
+    var ret :- expect client.GetBoolean(SimpleBoolean.Types.GetBooleanInput(value:= Some("TEST_SIMPLE_BOOLEAN_VALUE")));
+    expect ret.value.UnwrapOr("") == "TEST_SIMPLE_BOOLEAN_VALUE";
+    print ret;
+} */
```

Comment:
still need to update these

Reply:
FYI I'm working on https://github.com/smithy-lang/smithy-dafny/issues/298 right now, which will provide a standard way to generate some of these target language tests.